tests/bluetooth/tester/src/model_handler.c

```diff
@@ -1271,6 +1273,7 @@ static struct bt_mesh_elem elements[] = {
 				 BT_MESH_MODEL_CFG_CLI(&cfg_cli),
 				 BT_MESH_MODEL_HEALTH_SRV(&health_srv, &health_pub),
 				 BT_MESH_MODEL_HEALTH_CLI(&health_cli),
+			     BT_MESH_MODEL_DTT_SRV(&dtt_srv),
```

Comment:
Not needed since Scheduler doesn't extend Scene server?

Reply:
"When a Scheduler Server model is present on an element, the Scene Server model shall also be present on the same element." So right now Scheduler Srv is also adding Scene Srv to the composition data. And that Scene server needs a Generic DTT Server.